### ISSUE TITLE
Test adding a thank you message on donation form before donating

### ISSUE THREAD

**Issue by adamlofting** [mozilla/donate.mozilla.org#159]
In order to focus on / maintain the emotional aspect of the donation experience, we discussed saying 'thank you' before the donation process, on the first view of the form. 

E.g. 'Thank you for helping Mozilla... donate now...'

Related photo from planning session:

![ty_on_form](https://cloud.githubusercontent.com/assets/1901816/9118668/db2fbb2c-3c26-11e5-9592-298eb195e637.jpg)

**Comment by adamlofting** [comment]
Next step is to work on copy (and design if required) for this test.

**Comment by ScottDowne** [comment]
It's not clear to me how much work this is.

Is this just new copy on the first step?

**Comment by adamlofting** [comment]
@ScottDowne yes, this is just a language / content test. If there is any development work it will be minimal and front-end.

**Comment by edrushka** [comment]
Clarifying: Does the copy go on page 1 of the form? So it's like the ["additional info" variant] (https://github.com/mozilla/donate.mozilla.org/issues/550) except instead of giving them context, we're thanking them?

**Comment by adamlofting** [comment]
Yes, this was a more subtle test around language. I can't remember who proposed it, but I think it was as simple as testing phrases like 'thank you for supporting Mozilla' and 'thank you for donating' on the first step of the form to see if this helped with momentum. 

It could be a simpler design variant such as adding text somewhere like this:
<img width="671" alt="screenshot 2015-10-29 13 41 38" src="https://cloud.githubusercontent.com/assets/1901816/10820088/753bcef0-7e43-11e5-9565-271d0fc74ec9.png">

**Comment by edrushka** [comment]
Sweet. I'll make a design ticket for Jordan.

**Comment by edrushka** [comment]
Oops. I don't think I ever made this ticket. Sorry! :flushed:
@adamlofting is this still a test we want to run?

**Comment by adamlofting** [comment]
@edrushka yes, let's keep this in the plan. But it's not urgent as we have many other tests we can run while we do this design.

**Comment by edrushka** [comment]
Copy ticket: #974 
Design ticket #973 

Design won't be completed until Orlando or later since Jordan's on PTO.
